bin/run_composable-kernels.sh

```diff
@@ -261,36 +291,104 @@ if [ "${SelectedSuite}" == 'client-examples' ]; then
     exit 1
   fi
 
-  # Remove parentheses from directories to exclude
-  # These might be added when providing the array via commandline
-  DirsToExclude=$(sed 's/[()]//g' <<< "${CK_CLIENT_EXAMPLES_TO_EXCLUDE[@]}")
+  # Process directories to exclude
+  # Usage of here-string to avoid sub-shell and removal of potential parentheses
+  read -ra DirsToExclude <<< "${CK_CLIENT_EXAMPLES_TO_EXCLUDE//[()]/}"
```

Comment:
this is so nice btw, I didn't knew that the pattern substitution works on arrays as well

Reply:
Yep, found out by accident, but it's really handy.